pkg/webhook/clients/clients.go

```diff
@@ -12,6 +12,7 @@ import (
 	v1 "k8s.io/api/admissionregistration/v1"
 	"k8s.io/client-go/rest"
 
+	ctlnetwork "github.com/harvester/harvester-network-controller/pkg/generated/controllers/network.harvesterhci.io"
```

Comment:
In your commit, there has been benerated files like [pkg/generated/controllers/network.harvesterhci.io/factory.go](https://github.com/harvester/harvester/pull/6538/files#diff-6b91fccaf8eac94906aa45d10b15e5f726962e25dac7ddeee9a8cc975e3f49b9)
did you try to import them instead of from `harvester/harvester-network/controller`?

Reply:
@w13915984028 yes it has to be imported from "github.com/harvester/harvester/pkg/generated/controllers/network.harvesterhci.io/" instead of "github.com/harvester/harvester-network-controller/pkg/generated/controllers/network.harvesterhci.io/".Changed in other places too.Thanks